docs/root/configuration/http_filters/lua_filter.rst

```diff
@@ -403,3 +414,19 @@ __pairs()
 
 Iterates through every *metadata* entry. *key* is a string that supplies a *metadata*
 key. *value* is *metadata* entry value.
+
+.. _config_http_filters_lua_connection_wrapper:
+
+Connection object API
+---------------------
+
+secure()
```

Comment:
Thing for the future where I'm guessing people are going to ask for SSL/TLS connection properties in Lua, should we instead return an SSL/TLS object? This object could have no methods right now but the presence of the object or nil could indicate secure? Just thinking for the future. Thoughts?

Reply:
Seems like a good idea. Will update it.

Reply:
Updated.